x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/swimlane/translations.ts

```diff
@@ -0,0 +1,282 @@
+  'xpack.triggersActionsUI.components.builtinActionTypes.swimlaneAction.configureAPILabel',
+  { defaultMessage: 'Configure API' }
+);
+
+export const SW_CONNECTOR_TYPE_LABEL = i18n.translate(
+  'xpack.triggersActionsUI.components.builtinActionTypes.swimlaneAction.connectorType',
+  {
+    defaultMessage: 'Connector Type',
+  }
+);
+
+export const SW_FIELD_MAPPING_IS_REQUIRED = i18n.translate(
+  'xpack.triggersActionsUI.components.builtinActionTypes.swimlaneAction.mappingFieldRequired',
+  {
+    defaultMessage: 'Field mapping is required.',
+  }
+);
+
+export const EMPTY_MAPPING_WARNING_TITLE = i18n.translate(
+  'xpack.triggersActionsUI.components.builtinActionTypes.swimlaneAction.emptyMappingWarningTitle',
+  {
+    defaultMessage: 'This connector has missing field mappings',
+  }
+);
+
+export const EMPTY_MAPPING_WARNING_DESC = i18n.translate(
+  'xpack.triggersActionsUI.components.builtinActionTypes.swimlaneAction.emptyMappingWarningDesc',
+  {
+    defaultMessage:
+      'This connector cannot be selected because it is missing the required case field mappings. You can edit this connector to add required field mappings or select a connector of type Alerts.',
+  }
+);
+
+export const SW_REQUIRED_ALERT_SOURCE = i18n.translate(
+  'xpack.triggersActionsUI.components.builtinActionTypes.swimlaneAction.error.requiredAlertSource',
+  {
+    defaultMessage: 'Alert source is required.',
+  }
+);
+
+export const SW_REQUIRED_SEVERITY = i18n.translate(
+  'xpack.triggersActionsUI.components.builtinActionTypes.swimlaneAction.error.requiredSeverity',
+  {
+    defaultMessage: 'Severity is required.',
+  }
+);
+
+export const SW_REQUIRED_CASE_NAME = i18n.translate(
+  'xpack.triggersActionsUI.components.builtinActionTypes.swimlaneAction.error.requiredCaseName',
+  {
+    defaultMessage: 'Case name is required.',
+  }
+);
+
+export const SW_REQUIRED_CASE_ID = i18n.translate(
+  'xpack.triggersActionsUI.components.builtinActionTypes.swimlaneAction.error.requiredCaseID',
+  {
+    defaultMessage: 'Case ID is required.',
+  }
+);
+
+export const SW_REQUIRED_COMMENTS = i18n.translate(
+  'xpack.triggersActionsUI.components.builtinActionTypes.swimlaneAction.error.requiredComments',
+  {
+    defaultMessage: 'Comments are required.',
+  }
+);
+
+export const SW_REQUIRED_DESCRIPTION = i18n.translate(
+  'xpack.triggersActionsUI.components.builtinActionTypes.swimlaneAction.error.requiredDescription',
+  {
+    defaultMessage: 'Description is required.',
+  }
+);
+
+export const SW_REQUIRED_ALERT_ID = i18n.translate(
+  'xpack.triggersActionsUI.components.builtinActionTypes.swimlaneAction.error.requiredAlertID',
+  {
+    defaultMessage: 'Alert ID is required.',
+  }
+);
+
+export const SW_ALERT_SOURCE_TOOLTIP = i18n.translate(
+  'xpack.triggersActionsUI.components.builtinActionTypes.swimlaneAction.alertSourceTooltip',
+  {
+    defaultMessage: 'The index of the alert. Use {index} in Detections.',
+    values: { index: '{{context.rule.output_index}}' },
```

Comment:
We're interested in this one, as it seems to be the RAC alerts-as-data index name.  It appears that we are currently just offering the user that they _can_ use this mustache variable "in Detections", as opposed to providing it as the default value of the field.  We'd be really concerned if it was the default value of the field, offering it as a hint seems "safer".

But we're also interested in how this flow works.  We send an index name to swimlane, what are they going to do with it?  Just note it as a reference, or actively use it in API calls back to Kibana?  

It looks like we're settling on having the index names be determined on a per-space basis, so in theory we could provide an additional level of indirection by just referencing the kibana space, which then could be used to obtain the alerts-as-data index name (how is unclear, but we'll clearly need some kind of API like that in RAC).

Reply:
Also worth noting, it's going to be difficult / impossible to "migrate" mustache templates in action params, due to the lack of surgical parsing of the templates.  Once we've provided a mustache template to users, we have to support it forever, or potentially break the older connector's execution if we stop supporting it.

Reply:
One more thing - the message could probably be more precise.  Not sure everyone will know what "Detections" are - in theory this connector could be used by any alert, and someday connectors will be usable outside of alerting.  So thinking it should be something like `Use {index} in Security Solution Detection Rules.`

Reply:
@pmuellr I can understand why you are interested in this field, we offer to them to give back the index-name because they are querying `siem-signal-*` with the alertId to get back the alert details. We wanted to make sure that they got the correct data for the alertId because we know that is possible that `siem-signal-001` and `siem-signal-002` might have identical _id.

We did agree with you to not default the value but just to give a hint if they want to use it.  Thinking more about it, we might not be ready to give that kind of details yet and if that's a blocker to you to move forward. I think that's ok to hide it and we will bring it up in our RAC meeting so we can discuss it. And they can still do it the way they were doing it before with some kind of risk.

Reply:
I don't think it's a blocker - but it does seem perhaps premature.  And we worry about users making use of this, and then somehow we have to support it forever.

> we know that is possible that siem-signal-001 and siem-signal-002 might have identical _id.

Are `-001` and `-002` ILM suffixes, or the "namespace" suffixes?  Guessing the "namespace", in which case - this seems a bit problematic.  I suppose searches and aggs would work for `.siem-signal-*` (but I don't know!), but a GET with a specific `_id` would be ... indeterminate?  This seems like a problem with the way we've decided to partition the alerts as data indices ... like maybe the partition name needs to be part of the `_id`?

Is Swimlane planning on doing direct ES queries, or using the curated alerts-as-data APIs?  I think if the curated APIs, we should aim to have them use the Kibana space instead of the index name, since presumably the curated APIs will take this as a parameter (possibly implicit, in the path of the URL).  Assuming we plan on having the partitions be based on something in the Kibana space.

Reply:
As we discussed I am gonna remove the Alert source until we can do it properly.

Reply:
The `-001` and `-002` are ILM suffixes, detection engineer told us it can happen. I agree with you that it is premature when I offered, I. was only thinking about rules from security solution. @cnasikas, let's remove it for now.

Thank you @pmuellr to keep us on our feet ❤️